src/candela/util/vega/test/transform.js

```diff
@@ -9,6 +9,9 @@ test('vcharts.transform() without spec', t => {
   t.deepEqual(true, vcharts.transform(true), 'boolean transform should be identity');
   t.deepEqual(null, vcharts.transform(null), 'null transform should be identity');
 
+  let d = new Date();
```

Comment:
I don't think it makes a huge difference but I'd feel safer if this constructed a particular date, rather than using the current time to make one (i.e., the testing state shouldn't depend on the current time, or any other external state).